container-admin/Dockerfile_client

```diff
@@ -22,14 +21,13 @@ RUN add-apt-repository -y ppa:webupd8team/java \
     && echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections \
     && apt-get install -y oracle-java8-installer
 
-# setup cwltool to support Dockstore
-RUN pip install setuptools==34.2.0
-RUN pip install cwl-runner cwltool==1.0.20170217172322 schema-salad==2.2.20170222151604 avro==1.8.1
-
 # install Consonance services
-ENV consonance_version={{ CONSONANCE_BINARY_VERSION }}
+ENV consonance_version=cgp-2.0.0-alpha.16
```

Comment:
Replacing a variable with a hard-coded value seems like it will introduce extra work in the future.